includes/admin_settings_tab.php

```diff
@@ -0,0 +1,333 @@
+				<div class="settings-card">
+					<h3><strong><?php _e('Google Maps API Key', '12-step-meeting-list') ?></strong></h3>
+					<p class="no_space_after"><?php _e('Enter a key from Google authorizing use of their Map Services (<a href="https://developers.google.com/maps/documentation/javascript/get-api-key" target="_blank">get instructions here</a>).', '12-step-meeting-list') ?></p>
+					<form class="columns" method="post" action="<?php echo $_SERVER['REQUEST_URI'] ?>">
+						<?php wp_nonce_field($tsml_nonce, 'tsml_nonce', false) ?>
+						<div class="input">
+							<input type="text" name="tsml_add_google_maps_key" value="<?php echo $tsml_google_maps_key ?>" placeholder="Enter Google API key here">
+						</div>
+						<div class="btn">
+							<?php if (empty($tsml_google_maps_key)) { ?>
+								<input type="submit" class="button" value="<?php _e('Add', '12-step-meeting-list') ?>">
+							<?php } else { ?>
+								<input type="submit" class="button" value="<?php _e('Update', '12-step-meeting-list') ?>">
+							<?php } ?>
+						</div>
+					</form>
+				</div>
+							
+				<div class="settings-card">
+					<h3><strong><?php _e('Address Geocoding', '12-step-meeting-list') ?></strong></h3>
+					<p><?php _e('Code4Recovery is working on a new method for geocoding addresses. You can decide whether to use the old way, or whether to help us test the new way.', '12-step-meeting-list') ?></p>
+				<form method="post" action="<?php echo $_SERVER['REQUEST_URI'] ?>">
+					<?php wp_nonce_field($tsml_nonce, 'tsml_nonce', false) ?>
+					<select name="tsml_geocoding_method" onchange="this.form.submit()">
+						<option value="legacy" <?php selected($tsml_geocoding_method, 'legacy') ?>>
+							<?php _e('Legacy Method', '12-step-meeting-list') ?>
+						</option>
+						<?php if (!empty($tsml_google_maps_key)) { ?>
+							<option value="google_key" <?php selected($tsml_geocoding_method, 'google_key') ?>>
+								<?php _e('Use my Google API Key', '12-step-meeting-list') ?>
+							</option>
+						<?php } ?>
+						<option value="api_gateway" <?php selected($tsml_geocoding_method, 'api_gateway') ?>>
+							<?php _e('BETA - API Gateway - BETA', '12-step-meeting-list') ?>
+						</option>
+					</select>
+				</form>
+				<?php if (!empty($tsml_google_maps_key)) { ?>
+					<p>If you select "Use my Google API Key", then you <strong>must</strong> go into the Google Console and enable the geocode API for your key</p>
+				<?php } ?>
+				</div>
+			</div>
+		</div>
+	</div>
+
+	<div id="column3" class="col one-third">
+		<!-- Put About Us here -->
+		<div class="postbox" >
+			<div class="inside">
+				<h1><?php _e('About Us', '12-step-meeting-list') ?></h1>
+				<div class="settings-card">
+					<div class="c4r-logo" style="float:right; margin-top:-55px; margin-right: -10px;" >
+						<a href="https://code4recovery.org"><img src="/wp-content/plugins/12-step-meeting-list/assets/img/code4recovery.svg" alt="Code For Recovery" ></a>										
```

Comment:
The image display is weird with my theme, currently 2019:

<img width="1086" alt="image" src="https://user-images.githubusercontent.com/31904316/160261022-28096ed7-340e-495f-94ad-3664e989f02b.png">

Reply:
Something strange here. The theme should not affect this admin display.

Reply:
This is what I see at the top of the Settings tab with the 2019 theme:
![image](https://user-images.githubusercontent.com/57125427/160261621-38b3dc5f-a1d1-4bfd-b68d-0c8763c17b48.png)

Reply:
I'm up to 24 themes that I've tested against, and other than some small CSS issues which I was able to resolve with some code in the Additional CSS cutomizer, they all worked reasonable well. Especially now that I've added the tsml_assets() in the template files.